ArduPlane/GCS_Mavlink.cpp

```diff
@@ -791,13 +792,197 @@ MAV_RESULT GCS_MAVLINK_Plane::handle_command_int_do_reposition(const mavlink_com
     return MAV_RESULT_FAILED;
 }
 
+// these are GUIDED mode commands that are RATE or slew enabled, so you can have more powerful control than default controls.
+MAV_RESULT GCS_MAVLINK_Plane::handle_command_int_guided_slew_commands(const mavlink_command_int_t &packet)
+{
+
+
+  ::printf("GCS_MAVLINK_Plane::handle_command_int_guided_slew_commands .. slew packet recieved:%d\n",packet.command);
```

Comment:
These need to be removed, these are also debug.

Reply:
will fix on next attempt.